### DISCUSSION TITLE
Support for ppa / AppImage

### DISCUSSION THREAD

**Discussion by skjerns** [posit-dev/positron#4522] [Feedback]
I'm working on a VM without sudo rights. The IT admins told me that they can only install & update the package when a ppa would host it.

1. Is there any plan to add the positron `.deb` to a repository for easier install & updating?
2. Is there a plan to package positron as an AppImage so that it can be executed/installed without sudo?

**Comment by jmcphers** [comment]
Good question! We're interested in both of these, but neither are on our near term roadmap.
